core/domain/question_services_test.py

```diff
@@ -116,5 +120,69 @@ def test_update_question(self):
         model = question_models.QuestionModel.get(question_id)
         self.assertEqual(model.question_data, new_question_data)
         self.assertEqual(
-            model.question_data_schema_version, question_data_schema_version)
-        self.assertEqual(model.language_code, language_code)
+            model.question_data_schema_version,
+            self.question_data_schema_version)
+        self.assertEqual(
+            model.language_code, self.language_code)
+        self.assertEqual(
+            model.status, self.status)
+
+    def test_compute_summary_of_question(self):
+        question_summary = question_services.compute_summary_of_question(
+            self.question, self.owner_id)
+
+        self.assertEqual(question_summary.id, 'dummy')
+        self.assertEqual(
+            question_summary.question_html_data,
+            html_cleaner.clean(''))
```

Comment:
Why are we asserting that it's equal to the empty string? Better to add some unique, specific data to the State and test for that. Otherwise you might have wrong code, but the test may accidentally pass because if everything's generally empty then it might easily match an empty string by accident.

Ditto below.

Reply:
Done.